src/progress.rs

```diff
@@ -977,6 +988,41 @@ impl MultiProgress {
         pb
     }
 
+    /// Removes a progress bar.
+    ///
+    /// Remove the progress bar at the index. If the index is not valid,
+    /// nothing will happen.
+    pub fn remove(&self, index: usize) {
+      let state = self.state.write().unwrap();
+      let len = state.objects.len();
+      if index >= len {
+        return;
+      }
+
+      let msg = DrawMessage::Remove {
+        ts: Instant::now(),
+      };
+      self.tx.send((index, msg)).unwrap();
+
+      /*
```

Comment:
Why is this commented out code here? I don't want to check in dead code. ALso not sure if this is supposed to be there or not.

Reply:
Ah, my bad, should've deleted this before submitting the PR. I've updated this commit to actually remove this code.

Reply:
@mitsuhiko could you take another look at this?